modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -617,20 +618,80 @@ public Response getEvents(@HeaderParam("Accept") String acceptHeader, @QueryPara
         }
 
         String name = filterTuple[0];
-        String value = filterTuple[1];
+        String value;
 
-        if ("presenters".equals(name))
+        if (!requestedVersion.isSmallerThan(ApiVersion.VERSION_1_1_0)) {
+          // MH-13038 - 1.1.0 and higher support semi-colons in values
+          value = f.substring(name.length() + 1);
+        } else {
+          value = filterTuple[1];
+        }
+
+        if ("presenters".equals(name)) {
           query.withPresenter(value);
-        if ("contributors".equals(name))
+        } else if ("contributors".equals(name)) {
           query.withContributor(value);
-        if ("location".equals(name))
+        } else if ("location".equals(name)) {
           query.withLocation(value);
-        if ("textFilter".equals(name))
+        } else if ("textFilter".equals(name)) {
           query.withText("*" + value + "*");
-        if ("series".equals(name))
+        } else if ("series".equals(name)) {
           query.withSeriesId(value);
-        if ("subject".equals(name))
+        } else if ("subject".equals(name)) {
           query.withSubject(value);
+        } else if (!requestedVersion.isSmallerThan(ApiVersion.VERSION_1_1_0)) {
+          // add filters only available with Version 1.1.0 or higher
```

Comment:
add = additional?

Reply:
add = add filters to query

Anyway, I've changed this to "additional"

Fixed.